ftp/1.6/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,37 @@
+= FTP Connector - Mule 4
+:page-aliases: connectors::ftp/ftp-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for FTP (FTP Connector) provides access to files and folders on an FTP server. FTP connector operations manage file transfers by using File Transfer Protocol (FTP). The connector's main features include:
+
+* The ability to read files or fully list directory contents on demand.
+* Support for common FTP operations such as creating directories and copying, moving, renaming, and deleting files.
+* Support for locking files.
+* File matching functionality.
+* A design that is consistent with the File and FTP Connectors.
++
+The same set of operations is available using these two connectors, and they behave similarly.
+
+=== Representation of File Attributes
+
+Mule 4 provides a message structure of a payload and attributes in which the data moves around, making that payload a file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, and timestamps. For FTP Connector and SFTP Connector, these attributes are known as  _file attributes_.
```

Comment:
Should the start of the sentence read like this instead?
"Mule 4 provides the message structure..."

Also, I'm wondering if "a" should be "the" at the end of the first sentence:
"...making that payload the file content."